android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/login/LoginScreen.kt

```diff
@@ -19,6 +25,12 @@ import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.typography
 import vn.luongvo.kmm.survey.android.ui.theme.ComposeTheme
 import vn.luongvo.kmm.survey.android.ui.theme.White50
 
+private const val FirstPhaseDurationInMilliseconds = 800
+private const val StayPhaseDurationInMilliseconds = 500
+private const val LastPhaseDurationInMilliseconds = 700
+private const val BlurRadius = 25f
+private val LogoOffset = Offset(0f, -229f)
```

Comment:
@luongvo I'm not sure if we should apply CamelCase convention for all variables, or we should apply it for specific variables like Color or Dimen?

Reply:
@manh-t I found that all sample projects follow to use CamelCase in all compose components, even Composable screen 🤓

Reply:
tbh I don't like this convention at all since it will make us confusing with the existing Kotlin/ Android project when switching contexts a lot 😮‍💨

Reply:
- https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#singletons_constants_sealed-class-and-enum-class-values
- https://github.com/detekt/detekt/discussions/4078